packages/block-editor/src/components/inserter/tips.js

```diff
@@ -28,7 +33,79 @@ const globalTips = [
 	__( "Change a block's type by pressing the block icon on the toolbar." ),
 ];
 
-function Tips() {
+const contextualTips = {
+	css: createInterpolateElement(
+		__(
+			'CSS - You can visit the <a>the Customizer</a> to edit the CSS on your site.'
+		),
+		{
+			a: <a href="/customize.php?autofocus[section]=custom_css" target="_blank"/>
+		}
+	),
+
+	theme: createInterpolateElement(
+		__(
+			'theme - You can visit the <a>theme directory</a> to select a different design for your site.'
+		),
+		{
+			a: <a href="/themes.php" target="_blank"/>
+		}
+	),
+
+	plugin: createInterpolateElement(
+		__(
+			'plugin - You can visit the <a>plugin directory</a> to install additional plugins.'
+		),
+		{
+			a: <a href="/plugin-install.php" target="_blank"/>
+		}
+	),
+
+	header: createInterpolateElement(
+		__(
+			'header - You can visit the <a>the Customizer</a> to edit your logo and site title.'
```

Comment:
I think only "Customizer" should be linked (drop "the"). The title of the page it goes to is "Customizer < [site title]," and the other tips in the list also leave off "the" from the linked text. 

cc @davemart-in

Reply:
Yep, I like that. Sounds good to me!